RedHat-UX/red-hat-design-system
# docs: style tables tokenValue shortcodes

## What I did

1. Created `tokenValue` shortcode that prints the value of a given token.
2. Replaced hex codes and border px with tokens in footer docs style tables.


## Testing Instructions

1. Make sure the [DP footer docs](https://deploy-preview-1056--red-hat-design-system.netlify.app/elements/footer/style/) output same content as [footer docs on ux dot](https://ux.redhat.com/elements/footer/style/).

## Notes to Reviewers
